src/casanovoutils/summarize_mgf.py

```
@@ -0,0 +1,1077 @@
+                    seq = params.get("seq", "")
+                    if not seq:
+                        continue
+
+                    n_with_seq += 1
+
+                    # Peptide length
+                    try:
+                        parsed_seq, _ = pyteomics_proforma.parse(seq)
+                        length_counts[len(parsed_seq)] += 1
+                    except Exception:
+                        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)